CoreFoundation/String.subproj/CFString.c

```diff
@@ -6442,14 +6442,16 @@ reswtch:switch (ch) {
 		return true;
 	    }
 	case '1': case '2': case '3': case '4': case '5': case '6': case '7': case '8': case '9': {
-	    int64_t number = 0;
+            long long _number = 0;
```

Comment:
Why the renaming of the `number`?  That seems gratuitous.

Reply:
`number` is used further down as an `int64_t` but that is probably not needed as it only checks for `INT32_MAX` later on, I will update.